website/data/open-source/projects.yml

```diff
@@ -16,6 +16,9 @@ Developer Productivity & Tools:
   - meshery/meshery
   - nikochiko/autosaved
 
+Terminal-based Applications:
+  - Samyak2/toipe
+
```

Comment:
Can you please include this as a part of the larger "Developer Productivity and Tools" category itself currently?

Reply:
Sure. I did not add it there because it isn't a developer tool specifically. I'll move it there

Reply:
Done